pkgs/applications/misc/poptracker/default.nix

```diff
@@ -0,0 +1,64 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, util-linux
+, SDL2
+, SDL2_ttf
+, SDL2_image
+, openssl
+, which
+, libsForQt5
+, makeWrapper
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "poptracker";
+  version = "0.25.7";
+
+  src = fetchFromGitHub {
+    owner = "black-sliver";
+    repo = "PopTracker";
+    rev = "5f3b4b1008a157dd82ec83a3f3c8ab40bb1ded48";
+    hash = "sha256-UqGEhs2AY9AKYrItmky3tcvXu/dXnnPwMXKwZ3klXcU=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    SDL2
+    SDL2_ttf
+    SDL2_image
+    openssl
+    util-linux
+    makeWrapper
+  ];
+
+  buildPhase = ''
+    runHook preBuild
+    make native CONF=RELEASE
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    mkdir $out
+    cp build/linux-x86_64/poptracker $out
+    cp -r assets $out/assets
+    wrapProgram $out/poptracker --prefix PATH : ${lib.makeBinPath [ which libsForQt5.kdialog ]}
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Scriptable tracker for randomized games";
+    longDescription = ''
+      Universal, scriptable randomizer tracking solution that is open source. Supports auto-tracking.
+
+      PopTracker packs should be placed in `~/PopTracker/packs` or `./packs`.
+    '';
+    homepage = "https://github.com/black-sliver/PopTracker";
+    changelog = "https://github.com/black-sliver/PopTracker/releases/tag/v${version}";
+    license = licenses.gpl3;
```

Comment:
The use of `gpl3` is deprecated and you should use either `gpl3Only` or `gpl3Plus`.